compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__numbers__int_literals.snap

```diff
@@ -11,6 +10,8 @@ function go() {
   0b00001111;
   0o17;
   0xF;
+  1.6e5;
```

Comment:
Maybe add examples with more digits?
Also add examples with underscores if that makes any sense and check before-after in tests (I think the current formatter/agreement is to not touch anything but every-day notation)

Reply:
> ... check before-after in tests

is there an existing example that I could follow?

Reply:
https://github.com/gleam-lang/gleam/pull/1899/files#diff-eace0c72b35f221894649a2a178e1799e60899a428b0db25e1aeca46562c5a0dR1205-R1238